bot/exts/evergreen/error_handler.py

```diff
@@ -83,7 +83,11 @@ async def on_command_error(self, ctx: commands.Context, error: commands.CommandE
             return
 
         if isinstance(error, commands.NoPrivateMessage):
-            await ctx.send(embed=self.error_embed("This command can only be used in the server.", NEGATIVE_REPLIES))
+            await ctx.send(
+                embed=self.error_embed(
+                    f"This command can only be used in the server. Go to <#{Channels.community_bot_commands}> instead!",
+                    NEGATIVE_REPLIES)
+            )
```

Comment:
The parentheses are not formatted correctly, they should look like:
```suggestion
            await ctx.send(
                embed=self.error_embed(
                    f"This command can only be used in the server. Go to <#{Channels.community_bot_commands}> instead!",
                    NEGATIVE_REPLIES
                )
            )
```